tests/nlu/test_utils.py

```diff
@@ -32,7 +32,10 @@ def fake_model_dir(empty_model_dir):
 
 def test_relative_normpath():
     test_file = "/my/test/path/file.txt"
-    assert utils.relative_normpath(test_file, "/my/test") == "path/file.txt"
+    assert (
+        utils.relative_normpath(test_file, "/my/test").replace("\\", "/")
+        == "path/file.txt"
+    )
```

Comment:
this should be platform-agnostic without the `replace()`
```suggestion
    assert (
        assert Path(utils.relative_normpath(test_file, "/my/test")) == Path("path/file.txt")
    )